### PR TITLE
PCQM4Mv2 cfg to run pretrained GPS model

### PR DESCRIPTION
**Pull request by rampasek**
Added config file to run inference on PCQM4Mv2 using a pretrained GPS model.

To run the inference:
1. Download and unzip: https://www.dropbox.com/s/677clfz3cng8xsi/pretrained-GPS-pcqm4m.zip?dl=1
You should now have a folder `pretrained/pcqm4m-GPS+RWSE.medium` in the root of the project.
This is the last epoch checkpoint from the original run used for the GPS paper preprint. It was trained using a random split of the official `train` set to train and validation set, taking the official `valid` set as the test set. The official `test-dev` and `test-challenge` sets are not used.

2. Run: `python main.py --cfg configs/GPS/pcqm4m-GPS-inference.yaml`
Note: This will download and process the PCQM4Mv2 dataset first (~1h, when you run it for the first time), then we need to precompute Laplacian decompositions (~2h on 4 core CPU), and finally run the inference.

This checkpoint achieves `MAE: 0.0860` on what is here called the test set, which is the official `valid` set.

### CODE REVIEW

**Comment by rampasek** [comment]
Note, this PR also closes #6 original request.
